.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -3,7 +3,7 @@ name: publish-technical-documentation-release
 on:
   push:
     branches:
-      - v[0-9]+.[0-9]+.x
+      - release-v[0-9]+.[0-9]+.[0-9]+
```

Comment:
```suggestion
      - release-[0-9]+.[0-9]+.[0-9]+
```

Reply:
It looks like this didn't actually get incorporated, I'll open up a PR

Reply:
OH sorry about that. I thought I hit "commit suggestion". :(